cypress/e2e/insights-date-picker.cy.ts

```diff
@@ -1,8 +1,11 @@
+import 'cypress-network-idle'
+
 import { urls } from 'scenes/urls'
 
 describe('insights date picker', () => {
     beforeEach(() => {
         cy.visit(urls.insightNew())
+        cy.waitForNetworkIdle(300)
```

Comment:
This was much easier than mocking requests to `/query/` and `/query/:queryId`, which I couldn't get working.